inception-layer-docmetadata/src/main/java/de/tudarmstadt/ukp/inception/ui/core/docanno/sidebar/DocumentMetadataAnnotationSelectionPanel.java

```diff
@@ -97,7 +101,10 @@ public DocumentMetadataAnnotationSelectionPanel(String aId, IModel<Project> aPro
         detailPanel = aDetails;
         selectedLayer = Model.of();
 
-        add(createAnnotationList());
+        annotationsContainer = new WebMarkupContainer("annotationsContainer");
```

Comment:
Dont we normally have constants for ids?

Reply:
Introduced them.